Update service-bus-dotnet-hybrid-app-using-service-bus-relay.md

JacksReacher opened the pull request:
Updating the section around transportClientEndpointBehavior to indicate this is just a warning and not a blocking issue.

azurecla commented:
Hi __@artsterb__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

JacksReacher committed:
Update service-bus-dotnet-hybrid-app-using-service-bus-relay.md

Updating the section around transportClientEndpointBehavior to indicate this is just a warning and not a blocking issue.
